src/api/app/controllers/webui/search_controller.rb

```diff
@@ -117,8 +117,14 @@ def set_parameters
     @search_text = @search_text.delete("'[]\n")
 
     @search_what = []
-    @search_what << 'package' if params[:package] == '1'
-    @search_what << 'project' if params[:project] == '1' || !@search_issue
+    if switch_to_webui2?
+      @search_what << 'package' if params[:search_for].in?(['0', '2'])
+      @search_what << 'project' if params[:search_for].in?(['0', '1']) && !@search_issue
+    else
+      # FIXME: only for bento
```

Comment:
This comment should be `TODO: bento_only` to be consistent with the other comments of this nature.